app.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path')
+const express = require('express')
+const hbs = require('express-handlebars')
+const router = require('./routes/index.js')
+
+const app = express()
+
+app.set('port', process.env.PORT || 4444)
+
+app.engine('hbs', hbs({
+  defaultLayout: 'main',
+  defaultDir: path.join(__dirname, './', 'views/layouts'),
+  partialsDir: path.join(__dirname, './', 'views/partials'),
+  extname: 'hbs'
+}))
+
+app.set('view engine', 'hbs')
+app.set('views', path.join(__dirname, './', 'views'))
+
+app.use(router)
```

Comment:
I think (not 100% sure) that the router should come below the static file middleware